src/routes/safe/components/Apps/components/AddAppForm/AppUrl.tsx

```diff
@@ -28,22 +28,34 @@ export const appUrlResolver = createDecorator({
   },
 })
 
-export const AppInfoUpdater = ({ onAppInfo }: { onAppInfo: (appInfo: SafeApp) => void }): null => {
+type AppInfoUpdaterProps = {
+  onAppInfo: (appInfo: SafeApp) => void
+  setIsLoading: (isLoading: boolean) => void
```

Comment:
Can we rename this callback to `onLoading`?

Reply:
you mean `setIsLoading` by `onLoading`? I wouldn't be agreed with that, `onLoading` seems a callback name for when `onLoading` is occurring and here, we are setting the state explicitly

Reply:
What I'm trying to say is this name looks like a state setter. But it's not. It's a callback prop.

The actual state is being set in `AddApp`. There, this name makes sense.

In `AppInfoUpdaterProps`, it's a callback prop, it's not setting any state in `AppInfoUpdaterProps` itself. So  it's confusing.

Reply:
I see your point, but it looks weird to me, from my point of view when you call something `onLoading` is when you are reacting to that change, in this case, we are producing the state change. 

but I'll follow your suggestion.

Reply:
There's no contradiction. `AddApp` is indeed reacting to the onLoading event from `AppInfoUpdaterProps`, and sets the internal state based on the passed value.

Reply:
done!